conn/physic/units.go

```diff
@@ -1178,9 +1295,20 @@ const (
 	Celsius      Temperature = Kelvin
 
 	// Conversion between Kelvin and Fahrenheit.
-	ZeroFahrenheit  Temperature = 255372 * MilliKelvin
+	ZeroFahrenheit  Temperature = 255372222222 * NanoKelvin
 	MilliFahrenheit Temperature = 555555 * NanoKelvin
 	Fahrenheit      Temperature = 555555555 * NanoKelvin
+
+	maxTemperature Temperature = (1 << 63) - 1
+	minTemperature Temperature = -((1 << 63) - 1)
```

Comment:
I think we should set it to 0. For the same reason as with RelativeHumidity, there's no true negative temperature.

That said there can be "negative rate of change" when looking at a differential of the measure. In this case we should allow the range to exceed, but this is only because the type is used to store a different unit (K/s or %rH/s) than what it is supposed to.

I'm still ambivalent, but both should be coherent (either we support or we don't)

Reply:
ok, I was thinking of more of and delta scenario, but that could always be expressed as +ve, so I'll make 0K the limit for all units.